fast64_internal/f3d/f3d_gbi.py

```diff
@@ -3168,6 +3214,16 @@ class GbiMacro:
     _segptrs = False
     _ptr_amp = False
 
+    tag = GfxTag.NoTag
+    """Type: GfxTag. The tag's current use is to determine how to write gfx scrolling code for this given command."""
+
+    fMaterial = None
```

Comment:
I can see this being an issue with having hashing on GbiMacro instances?

Reply:
Would implementing id() based hashing fix that issue?

Reply:
Bureauctopus/Jesusyoshi said that without annotations, these wouldn't be "part" of the dataclass so not used for the hash (and equality comparison I assume)

so it apparently wouldn't be an issue, but worth making sure and documenting

Reply:
I actually tried adding annotations, but I got an error about constructor arguments related to dataclasses. I'll add some comments on it not being part of the hash.